pwiz_tools/Skyline/ToolsUI/PythonInstallerLegacyDlg.Designer.cs

```diff
@@ -1,92 +1,92 @@
-        /// </summary>
-        private void InitializeComponent()
-        {
-            System.ComponentModel.ComponentResourceManager resources = new System.ComponentModel.ComponentResourceManager(typeof(PythonInstaller));
-            this.btnInstall = new System.Windows.Forms.Button();
-            this.btnCancel = new System.Windows.Forms.Button();
-            this.labelMessage = new System.Windows.Forms.Label();
-            this.clboxPackages = new System.Windows.Forms.CheckedListBox();
-            this.SuspendLayout();
-            // 
-            // btnInstall
-            // 
-            resources.ApplyResources(this.btnInstall, "btnInstall");
-            this.btnInstall.DialogResult = System.Windows.Forms.DialogResult.Cancel;
-            this.btnInstall.Name = "btnInstall";
-            this.btnInstall.UseVisualStyleBackColor = true;
-            this.btnInstall.Click += new System.EventHandler(this.btnInstall_Click);
-            // 
-            // btnCancel
-            // 
-            resources.ApplyResources(this.btnCancel, "btnCancel");
-            this.btnCancel.DialogResult = System.Windows.Forms.DialogResult.Cancel;
-            this.btnCancel.Name = "btnCancel";
-            this.btnCancel.UseVisualStyleBackColor = true;
-            // 
-            // labelMessage
-            // 
-            resources.ApplyResources(this.labelMessage, "labelMessage");
-            this.labelMessage.Name = "labelMessage";
-            // 
-            // clboxPackages
-            // 
-            resources.ApplyResources(this.clboxPackages, "clboxPackages");
-            this.clboxPackages.CheckOnClick = true;
-            this.clboxPackages.FormattingEnabled = true;
-            this.clboxPackages.Name = "clboxPackages";
-            // 
-            // PythonInstaller
-            // 
-            this.AcceptButton = this.btnInstall;
-            resources.ApplyResources(this, "$this");
-            this.AutoScaleMode = System.Windows.Forms.AutoScaleMode.Font;
-            this.CancelButton = this.btnCancel;
-            this.Controls.Add(this.clboxPackages);
-            this.Controls.Add(this.labelMessage);
-            this.Controls.Add(this.btnCancel);
-            this.Controls.Add(this.btnInstall);
-            this.MaximizeBox = false;
-            this.MinimizeBox = false;
-            this.Name = "PythonInstaller";
-            this.ShowIcon = false;
-            this.ShowInTaskbar = false;
-            this.Load += new System.EventHandler(this.PythonInstaller_Load);
-            this.ResumeLayout(false);
-
-        }
-
-        #endregion
-
-        private System.Windows.Forms.Button btnInstall;
-        private System.Windows.Forms.Button btnCancel;
-        private System.Windows.Forms.Label labelMessage;
-        private System.Windows.Forms.CheckedListBox clboxPackages;
-    }
+﻿namespace pwiz.Skyline.ToolsUI
+{
+    partial class PythonInstallerLegacyDlg
```

Comment:
Lots of whitespace changing in the files associated with PythonInstallerLegacyDlg. Turning off whitespace diffs the changes look much more like the simple rename expected.